ntoskrnl/ex/locale.c

```diff
@@ -24,8 +24,250 @@ LANGID PsInstallUILanguageId = LANGIDFROMLCID(0x00000409);
 LCID PsDefaultThreadLocaleId = 0x00000409;
 LANGID PsDefaultUILanguageId = LANGIDFROMLCID(0x00000409);
 
+/* DEFINES *******************************************************************/
+
+#define BOGUS_LOCALE_ID 0xFFFF0000
+#define LCID_HEX_FORMAT_DIGIT 8
+
 /* PRIVATE FUNCTIONS *********************************************************/
 
+/**
+ * @brief
+ * Validates the registry data of a NLS locale.
+ *
+ * @param[in] LocaleData
+ * A pointer to partial information that contains
+ * the NLS locale data.
+ *
+ * @return
+ * Returns TRUE if the following conditions are met,
+ * otherwise FALSE is returned.
+ */
+static
```

Comment:
It may be of interest to inline this function. Or just put its code directly within ExpValidateNlsLocaleId since it's used only there.

Reply:
Opted for `__inline`.